CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 
 ### Minor
 
+- Icon: Add phone icon (#746)
```

Comment:
@anyi-diao Thank you for adding this icon! Moving forward, we should add these changes to the `## [Unreleased]` section. This time, we'll do that for you.

![image](https://user-images.githubusercontent.com/127199/76541162-61388500-6440-11ea-8c4a-137744287420.png)